src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -1157,10 +1168,11 @@ export default class MutationTable<
         };
     }
 
-    @computed protected get orderedColumns(): MutationTableColumnType[] {
-        const columns = (this.props.columns || []) as MutationTableColumnType[];
+    @computed protected get orderedColumns(): MutationTableColumnTypes[] {
+        const columns = (this.props.columns ||
+            []) as MutationTableColumnTypes[];
```

Comment:
Don't think you need this cast.
```suggestion
        const columns = (this.props.columns || []);
```